classes/templates.php

```diff
@@ -107,10 +107,10 @@ function get_template_structure() {
 				// return post_content
 			} else {
 				$return = "<div class='sm-single-location-default-template'>";
-				$return .= "<div class='sm-single-map'>[sm-location data='iframe-map' map_width='100px' map_height='100px']<a href=\"[sm-location data='directions']\" target=\"_blank\"></a></div>";
-				$return .= "<div class='sm-single-location-data'>[sm-location data='full-address']";
-				$return .= "<br /><a href=\"[sm-location data='directions']\" target=\"_blank\">Get Directions</a>";
-				$return .= "<ul class='sm-single-location-data-ul'>[sm-location data='phone' before='<li>' after='</li>'] [sm-location data='email' before='<li><a href=\"mailto:%self%\">' after='</a></li>'] [sm-location data='sm_category' format='csv' before='<li>Categories: ' after='</li>'] [sm-location data='sm_tag' format='csv' before='<li>Tags: ' after='</li>']</ul>";
+				$return .= "<div class='sm-single-map'>[sm-location data='iframe-map']<a href=\"[sm-location data='directions']\" target=\"_blank\"></a></div>";
+				$return .= "<div class='sm-single-location-data'><div id=\"sm-address\">[sm-location data='full-address']</div>";
+				$return .= "<div id=\"sm-directions\"><a href=\"[sm-location data='directions']\" target=\"_blank\">Get Directions</a></div><br />";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line exceeds 120 characters; contains 148 characters

<!-- Comment by [SideCI](https://sideci.com) -->